humble/distribution.yaml

```diff
@@ -3219,6 +3219,19 @@ repositories:
       version: master
     status: developed
   metrics_msgs:
+    doc:
+      type: git
+      url: https://github.com/MetroRobots/metrics_msgs.git
+      version: main
+    release:
+      packages:
+      - benchmark_msgs
+      - benchmark_utils
```

Comment:
Can you please elaborate on the functionality of these packages, what exactly does it aim to benchmark?

Reply:
Compute time, primarily. Do you think the README needs further clarification?

Reply:
README is fine, but benchmark term could mean a lot of things to different people. It seems overly generic, and not following guidelines 1 and 2 in [REP#144](https://www.ros.org/reps/rep-0144.html)

Reply:
I will also agree that these package names are too generic for rosdistro. I might suggest `metronav_metrics_msgs` (etc.)

Reply:
Ok, how about we call the whole thing `metro_interfaces` with `compute_benchmarking_msgs`, `compute_benchmarking` and `collision_msgs`

Reply:
The repository name isn't the concern, it's the actual package names that turn into e.g. `ros-jazzy-collision-msgs`, right. The generic package names suggest a generic use case. But what is a Collision here? It's defined in the context of 2D AMR Navigation, with a concern for events and entities. My first instinct was to expect a 3d normal vector and force, coming from a physical simulation perspective. However if you call the package `metronav_collision_msgs` then there's no confusion and no claim to some arbitrary generality, it's attached to a project with a focus on a use case - but that doesn't stop anybody from using it in their projects.

I realize maybe this is getting overly nitpicky, it's been a common topic recently and I think it's a really fuzzy line of "when does a package get to use a generic name like sensor_msgs" - though maybe one heuristic I have a feeling about is like "generic means community-approved standards, and standards means a REP"?

Reply:
Yeah, I get it. I'm not trying to standardize everyone, but I do try to name things in ways that if they were to become standards, the name would still be sufficiently generic. I mean, I'd like to try to build a common msg structure that could be something in common between [`gazebo_msgs/ContactState.msg`](https://github.com/ros-simulation/gazebo_ros_pkgs/blob/ros2/gazebo_msgs/msg/ContactState.msg) and [`moveit_msgs/CollisionObject.msg`](https://github.com/ros-planning/moveit_msgs/blob/ros2/msg/CollisionObject.msg), but that process is likely a REP can of worms, and really, all I want is to have some data recorded into a bag in a generic way. So I could just have `dlu_msgs/ComputeTime` and `dlu_msgs/Collisions`, but that doesn't encourage anyone else to pick up the standard. (I still have no idea who hector is of the 40some `hector_` prefixed packages. )

I'm willing to play ball and rename things as needed, but I'd like to find the middle-ground between company-branded specific names like `metro_collision_msgs` and overly generic package names (which is what I'm trying to do over on https://github.com/ros/rosdistro/pull/38370#issuecomment-1707509852 ). My two proposals are 
 * `compute_benchmarking_msgs` and `compute_benchmarking`
 * `base2d_collision_msgs`

Reply:
yeah, in the interest of unblocking, that seems to at least narrow in the package scope somewhat - let's move forward with that proposal

Reply:
@emersonknapp Can you please review https://github.com/MetroRobots/metrics_msgs/pull/3

Reply:
@emersonknapp I would much rather go through the approval process for the proper naming of packages at the repo level than in the rosdistro stage. Can you please take a look at https://github.com/MetroRobots/metrics_msgs/pull/3